### ISSUE TITLE
tzdata download URL broken (currently?)

### ISSUE THREAD

**Issue by rbvermaa** [JuliaTime/TimeZones.jl#180]
Our builds that depend on TimeZones.jl started failing due to the inability to download the tzdata. The server returns a 502 Bad gateway error. As this is an external dependency, we cannot do anything until the people who own the website fix the URL.

It seems that downloading from HTTP in stead of HTTPS does work at the moment, so we are using the workaround here to make our builds work again until the issue gets resolved. https://github.com/RelationalAI-oss/TimeZones.jl/commit/a586cef12b42035ece23b52709729ab1d94e37a4

**Comment by rbvermaa** [comment]
Somehow it looks like they only serve the newest tzdata on https:

```
$ curl --fail -O https://data.iana.org/time-zones/releases/tzdata2018i.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  368k  100  368k    0     0   403k      0 --:--:-- --:--:-- --:--:--  403k

$ curl --fail -O https://data.iana.org/time-zones/releases/tzdata2018e.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:05 --:--:--     0
curl: (22) The requested URL returned error: 502 
```

Using http in stead of https works for both links.

**Comment by Benardi** [comment]
Same problem using Debian 9, can't use Genie.jl properly for this reason. Seems to be working under Ubuntu.

**Comment by omus** [comment]
Thanks for reporting this. Interesting that `http` is working more reliably. I can switch the default over to using HTTP instead of HTTPS but I expect this problem is just with the IANA servers.

**Comment by omus** [comment]
So looking more closely it appears that I'm using a link that redirects. I'll update the URLs so that no redirects are required. I'll also note that the HTTP link does redirect to HTTPS so it's possible that the URL was just using a failed redirect.

I'll have a PR for this shortly.

**Comment by omus** [comment]
Should be fixed in v0.8.5 (https://github.com/JuliaLang/METADATA.jl/pull/20816)
